extern double NAV_RATE;
extern double GUIDE_RATE;

extern double SCSIM_ATT_ERROR;
extern double SCSIM_POS_ERROR;
extern double SCSIM_ATT_RATE;
extern double SCSIM_POS_RATE;

extern double TEST_ERROR;
extern double TEST_RATE;


extern double SMALL_ERROR;
extern double SMALL_VELOCITY;
extern double BIG_VELOCITY;
extern double KP_DEFAULT;
extern double KD_DEFAULT;
extern double KP_BIG;
extern double KD_BIG;
